### PR TITLE
public method count

### PR DESCRIPTION
**Pull request by M-Aarsman**

### CODE REVIEW

File: scripts/summarize.sh [hunk 1]
```diff
@@ -47,6 +47,8 @@ echo "3 (Builder, Factory Method in Shared module, and MVC in Client module - si
 
 echo "[ Q ] How many public APIs do we have.."
 echo "3: communication, conversion and fleet in Shared module"
+echo "public methods: "
```
Comment (korotkevics) [inline scripts/summarize.sh]: please replace three lines above with:
 echo "[ Q ] How many public APIs do we have..(=public method count)"

Comment (M-Aarsman) [reply scripts/summarize.sh]: ok

File: scripts/summarize.sh [hunk 2]
```diff
@@ -47,6 +47,8 @@ echo "3 (Builder, Factory Method in Shared module, and MVC in Client module - si
...
+grep -r "public .*) {*" --include=*.java . --no-filename | sort -u |wc -l
 
 echo "[ Q ] How many package-private APIs do we have.."
-echo "2: event triggers, message handlers in Client module"
\ No newline at end of file
+echo "2: event triggers, message handlers in Client module"
```
Comment (korotkevics) [inline scripts/summarize.sh]: Why did you add this?
